File: 06.js
numeral = Number(prompt('Ingresa un número: '));

const resultado = 1;

function factorial(num) {
  if (num === 0) {
    return 1;
  }
  return num * factorial(num - 1);
}

console.log(`El factorial de ${numeral} es ${factorial(numeral)}`);

File: 06.py
numeral = int(input('Ingresa un número: '))

resultado = 1


def factorial(numeral):
    if numeral == 0:
        return 1
    else:
        return numeral * factorial(numeral-1)


print(f'El factorial de {numeral} es {factorial(numeral)}')
